via/db.py

```diff
@@ -0,0 +1,57 @@
+import alembic.command
+import alembic.config
+import zope.sqlalchemy
+from sqlalchemy import MetaData, create_engine, text
+from sqlalchemy.exc import ProgrammingError
+from sqlalchemy.orm import DeclarativeBase, MappedAsDataclass, Session
+
+
+class Base(MappedAsDataclass, DeclarativeBase):
+    metadata = MetaData(
+        naming_convention={
+            "ix": "ix_%(column_0_N_label)s",
+            "uq": "uq_%(table_name)s_%(column_0_N_name)s",
+            "ck": "ck_%(table_name)s_%(constraint_name)s",
+            "fk": "fk_%(table_name)s_%(column_0_N_name)s_%(referred_table_name)s_%(referred_column_0_N_name)s",
+            "pk": "pk_%(table_name)s",
+        }
+    )
+
+
+def stamp(engine):  # pragma: no cover
+    """If the DB isn't already stamped then stamp it with the latest revision."""
+    stamped = False
+
+    with engine.connect() as connection:
+        try:
+            if connection.execute(text("select * from alembic_version")).first():
+                stamped = True
+        except ProgrammingError:
+            pass
+
+    if not stamped:
+        alembic.command.stamp(alembic.config.Config("conf/alembic.ini"), "head")
```

Comment:
This is copy-pasted from other projects and used in dev only (so I don't think it's worth unit testing), it just automatically stamps the DB with the latest Alembic revision in dev, so that people don't have to stamp it manually (which is a PITA)